build.sbt

```diff
@@ -228,7 +228,7 @@ def CiCommand(name: String)(commands: List[String]): Command =
       case (state, command) => ci(command) :: state
     }
   }
-def ci(command: String) = s"plz ${sys.env("CI_SCALA_VERSION")} $command"
+def ci(command: String) = s"++ ${sys.env("CI_SCALA_VERSION")} $command"
```

Comment:
sbt-doge's `plz <version> <command>` migrated into `++ <version>! <command>` once. https://github.com/sbt/sbt/pull/2613
As sbt 1.2.0, `plz` seems to be compatible with `++` https://github.com/sbt/sbt/issues/3698

Reply:
One of my favorite parts of sbt 1.0 is that sbt-doge is no longer needed 😄 (at least not for the basic use-case I had in the scalafmt build)